app/views/inthenewspoets/items/show.html.erb

```diff
@@ -19,10 +19,11 @@
   <% birthplace = @res["spatial"].select { |place| place["type"] == "birth place" } %>
   <% if birthplace %>
     <div>
-      Birth location: <%= birthplace[0]["country"] %>
+      Birth location: 
       <% if birthplace[0]["city"] %>
-        , <%= birthplace[0]["city"]  %>
+        <%= birthplace[0]["city"]  %>, 
       <% end %>
+      <%= birthplace[0]["country"] %>
```

Comment:
This looks good assuming country is always populated. If not, then the logic would need adjusted for whether a comma is displayed or not

Reply:
I don't think there was ever a case where city but not country was populated (in fact, I'm not sure city is populated in the current Airtable). We wouldn't want to display city only, I don't think. It might be better to put a conditional around the whole thing.

Reply:
Looking at the data ingest code, country will always be populated if there is anything for the birthplace.